.circleci/config.yml

```diff
@@ -278,30 +290,25 @@ jobs:
               #TODO: instead of reinstall, do this with configuration
               ./build.sh --headless
               pytest --cov-report=xml --cov=./ --cov-append tests/test_physics.py tests/test_sensors.py
+
+              # Generate JS CodeConv
+              npm run test_with_coverage
```

Comment:
Maybe, makes sense to have a separate step for JS testing?

Reply:
This is inside `run sim tests` so it should be good to keep all tests in same place as it runs CPP as well as Python tests? We do have a separate step also `js_lint_and_test` which tests without code coverage.